spec/jekyll-feed_spec.rb

```diff
@@ -249,6 +249,12 @@
     end
   end
 
+  context "feed stylesheet" do
```

Comment:
Could you provide a little more direction on this one? I believe I was able to tackle the other two refactorings.